src/main/scala/com/fulcrumgenomics/umi/GroupReadsByUmi.scala

```diff
@@ -482,6 +514,30 @@ class GroupReadsByUmi
     }
   }
 
+  /** When a minimum UMI length is specified, truncates all the UMIs to the length of the shortest UMI.  For the paired
+    * assigner, truncates the first UMI and second UMI sepeartely.*/
+  private def truncateUmis(umis: Seq[Umi]): Seq[Umi] = this.minUmiLength match {
+    case None => umis
+    case Some(_minUmiLength) =>
```

Comment:
I think having `minUmiLength` and `_minUmiLength` is confusing.  How about just `len` or `minLength`?